sdk/lib/opentelemetry/sdk/trace/samplers.rb

```diff
@@ -37,18 +37,18 @@ module Trace
       #   to the {Span} to be created. Can be nil.
       # @return [Result] The sampling result.
       module Samplers
-        RECORD_AND_SAMPLED = Result.new(decision: Decision::RECORD_AND_SAMPLED)
-        NOT_RECORD = Result.new(decision: Decision::NOT_RECORD)
-        RECORD = Result.new(decision: Decision::RECORD)
-        SAMPLING_HINTS = [Decision::NOT_RECORD, Decision::RECORD, Decision::RECORD_AND_SAMPLED].freeze
+        RECORD_AND_SAMPLE = Result.new(decision: Decision::RECORD_AND_SAMPLE)
+        IGNORE = Result.new(decision: Decision::IGNORE)
```

Comment:
Should `Ignore` be `Drop`?

Reply:
Thanks - I missed https://github.com/open-telemetry/opentelemetry-specification/pull/956 (and pasted the wrong link again 🤦 ).

Reply:
Fixed